src/gui/search/pluginselectdlg.cpp

```diff
@@ -424,10 +424,10 @@ void PluginSelectDlg::checkForUpdatesFinished(const QHash<QString, PluginVersion
         return;
     }
 
-    foreach (const QString &pluginName, updateInfo.keys()) {
+    for (auto i = updateInfo.begin(); i != updateInfo.end(); ++i) {
```

Comment:
possible to use `cbegin`, `cend`?

Reply:
Also please check other places (where you used `begin()`, `end()`) if `cbegin()`, `cend()` is applicable there.

Reply:
updateInfo is const.

Reply:
>updateInfo is const.

I don't really push hard for this, I mean why not unify all usages as there are no downsides using `cbegin()` when possible.

Reply:
`updateInfo` is `const`. `i` will be of `const_iterator` type in both cases.
In a previous PR review, in a similar situation, I used `cbegin()`/`cend()` and was asked to use `begin()`/`end()`.
I'm Ok with both ways. To save time, maybe an accepted one should be defined.

Reply:
Then it should be the same as in other cases. If we are not going to change something, it should be declared as a constant. In this case, if you don't need to modify it, then use cbegin/cend.

Reply:
@Chocobo1 I do agree with that. That was my reasoning. But I was asked to change it to `begin/end` because the container was `const` and it would result in the same `const_iterator`.

@glassez That's I what I did in another  PR. And I was keeping it consistent. I didn't use `cbegin/cend` to avoid being asked to change it to `begin/end`.

Will do it.

Reply:
>@glassez That's I what I did in another PR. And I was keeping it consistent. I didn't use cbegin/cend to avoid being asked to change it to begin/end.

The reviewer may not see any aspects, so you do not need to blindly follow all the advice (unless it points to some rule). We are always open to counter-arguments. I'm sorry if I forced you to do an extra job.